static/install.sh

```diff
@@ -123,3 +158,22 @@ else
 fi
 
 
+curl -X POST https://api2.amplitude.com/2/httpapi \
+  -H 'Content-Type: application/json' \
+  -H 'Accept: */*' \
+  -s \
+  -o /dev/null \
+  --data-binary @- << EOF
+  {
+      "api_key": "751db5fc75ff34f08a83381f4d54ead6",
```

Comment:
Can you move the api key to a variable and use a new one here - instead of the same one we use in the cli?